event_handling/src/Worker.cpp

```diff
@@ -1,76 +1,77 @@
 #include "essentials/Worker.h"
-#include "essentials/ITrigger.h"
 #include "essentials/Timer.h"
 
 #include <string>
+#include <thread>
 
 namespace essentials
 {
 
-Worker::Worker(std::string name)
```

Comment:
this class isn't adding any value on top of timer. other than making it even more bloated with the addition of another thread. maybe can get rid of it

Reply:
The Idea was to have something like a Behaviour from the engine ready to use outside of ALICA. I used it in some of my evaluations of my thesis. If you want to get rid of it, I will copy that to some project in a new git-repo on my account.